app/javascript/application.js

```diff
@@ -1,4 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
-import * as bootstrap from "bootstrap"
+import '@hotwired/turbo-rails'
+import 'controllers'
+import * as bootstrap from 'bootstrap' // eslint-disable-line no-unused-vars
```

Comment:
Invoking the JS linter outside the context of `yarn run`, it found additional violations. That and the fact that we don't need yarn scripts to run linters motivated me to change our approach to JS linting ever-so-slightly.